tests/unit/lms/services/lti_outcomes_test.py

```diff
@@ -160,27 +166,19 @@ def test_requests_fail_and_no_description_returned(self, svc, respond_with):
         with pytest.raises(LTIOutcomesAPIError, match="LTI outcome request failed"):
             svc.read_result(self.GRADING_ID)
 
-    def test_it_gracefully_handles_RequestException(self, requests, svc):
-        requests.post.side_effect = RequestException
-
-        with pytest.raises(
-            LTIOutcomesAPIError, match="Error calling LTI Outcomes service"
-        ):
-            svc.read_result(self.GRADING_ID)
-
     @classmethod
-    def sent_body(cls):
-        return xmltodict.parse(httpretty.last_request().body)
+    def sent_body(cls, http_service):
```

Comment:
I added `http_service` as an argument to this classmethod and the others below, which means the tests that call these classmethods now need to pass in `http_service`. Alternatively I could have turned the classmethods into fixtures that could get `http_service` themselves using fixture injectin